graphs/bellman_ford.py

```diff
@@ -1,56 +1,53 @@
-from __future__ import annotations
+from typing import Dict, List
 
 
-def printDist(dist, V):
-    print("Vertex Distance")
-    distances = ("INF" if d == float("inf") else d for d in dist)
-    print("\t".join(f"{i}\t{d}" for i, d in enumerate(distances)))
+def print_distance(distance: List[float], src):
+    print(f"Vertex\tShortest Distance from vertex {src}")
+    for i, d in enumerate(distance):
+        print(f"{i}\t\t{d}")
 
 
-def BellmanFord(graph: list[dict[str, int]], V: int, E: int, src: int) -> int:
+def bellman_ford(
+    graph: List[Dict[str, int]], vertex_count: int, edge_count: int, src: int
+) -> List[float]:
     """
     Returns shortest paths from a vertex src to all
     other vertices.
+    >>> edges = [(2, 1, -10), (3, 2, 3), (0, 3, 5), (0, 1, 4)]
```

Comment:
Why not just create `test_edges` at global scope and then use them here and also at line 46.

Reply:
Will look into it

Reply:
Now, `edges` is being used only here. So, it has not been declared globally.